developers/weaviate/model-providers/anthropic/generative.md

```diff
@@ -236,6 +258,29 @@ In other words, when you have `n` search results, the generative model generates
 
 </Tabs>
 
+### RAG with images
+
+You can also use images when querying Weaviate and perform retrieval augmented generation in both single prompts and grouped tasks. 
+
+<Tabs groupId="languages">
+
+ <TabItem value="py" label="Python API v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START WorkingWithImagesAnthropic"
```

Comment:
I think these tags don't need suffixes (applies to multiple similar blocks for other providers)

(e.g. should maybe be `# START WorkingWithImages`)?

Reply:
Not sure how to change this because each of the providers calls a different method (`GenerativeConfig.anthropic`)

Reply:
Oh I'm sorry. Yeah you are right. Then maybe the example was missing when I looked at it.